config/locales/en.yml

```diff
@@ -621,6 +633,19 @@ en:
       applications: *applications
       applications_tooltip: Number of open applications
 
+  payments:
+    create:
+      page_title: &page_title '%{payment_type} Payment by: %{user_name}'
+      title: *page_title
+      something_wrong: Something went wrong - please contact system administrator
```

Comment:
Need to add the email address: medlem@sverigeshundforetagare.se
You have a mail address in the SV version, but not here.